subprojects/core/src/main/java/org/gradle/caching/configuration/internal/DefaultBuildCacheConfiguration.java

```diff
@@ -110,6 +112,17 @@ public void remote(Action<? super BuildCache> configuration) {
         configuration.execute(remote);
     }
 
+    @Nonnull
+    @Override
+    public Set<BuildCacheServiceRegistration> getRegistrations() {
+      return registrations;
+    }
+
+    @Override
+    public void setRegistrations(@Nonnull Iterable<BuildCacheServiceRegistration> registrations) {
```

Comment:
It would be ok to accept a `Set` here and simply take ownership.

Reply:
Thanks I'll do that